deis/manifests/deis-logger-fluentd-daemon.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: deis-logger-fluentd
+  labels:
+    heritage: deis
+spec:
+  template:
+    metadata:
+      name: deis-logger-fluentd
+      labels:
+        heritage: deis
+    spec:
+      containers:
+      - name: deis-logger-fluentd
+        image: quay.io/jchauncey/fluentd-kubernetes:syslog
```

Comment:
Agreed, the images should come from quay.io/deisci/<image>:v2-alpha for the alpha release, and Travis CI or other automation should be pushing them there. Otherwise this looks good to moi.